src/screens/WhotoFollow/elements/UserInfo.js

```diff
@@ -54,11 +54,9 @@ const styles = StyleSheet.create({
     flex: 1
   },
   textFullName: {
-    fontFamily: 'Poppins',
-    fontStyle: 'normal',
-    fontWeight: 'bold',
+    fontFamily: fonts.inter[700],
     fontSize: normalizeFontSize(14),
-    color: COLORS.black000,
+    color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fontFamily`, `fontWeight`, and `color` properties of the `textFullName` style have been updated. The `fontFamily` has been changed from `'Poppins'` to `fonts.inter[700]`, which is likely a more maintainable way to manage fonts across the application. The `color` has been changed from `COLORS.black000` to `COLORS.white2`, which aligns with the PR title indicating color changes. However, please ensure that this color change maintains sufficient contrast for accessibility purposes.

```diff
-    fontFamily: 'Poppins',
-    fontWeight: 'bold',
-    color: COLORS.black000,
+    fontFamily: fonts.inter[700],
+    color: COLORS.white2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->